megatron/checkpointing.py

```diff
@@ -222,14 +222,17 @@ def save_checkpoint(iteration, model, optimizer, opt_param_scheduler):
     if not args.deepspeed:
         model = unwrap_model(model)
 
-    print_rank_0('saving checkpoint at iteration {:7d} to {}'.format(
```

Comment:
Why remove the formatting?

Reply:
Because the name of the iteration is a string 'release' instead of an integer value, do you think naming it as 'release' is more appropriate and reasonable? or should I make it as 0 iteration?

Reply:
Apologies for the late response. I think `release` is a reasonable iteration identifier.